datanode/sqlstore/market_data_test.go

```diff
@@ -892,7 +892,7 @@ func setupMarketData(t *testing.T, ctx context.Context) (*sqlstore.MarketData, e
 		marketData := csvToMarketData(t, line, seqNum)
 		seqNum++
 
-		if _, alreadyAdded := addedBlocksAt[marketData.VegaTime]; !alreadyAdded {
+		if _, alreadyAdded := addedBlocksAt[marketData.VegaTime.UnixMicro()]; !alreadyAdded {
```

Comment:
why the use UnixMicro here? everywhere else we do Nano. Not that it matter much but eh.

Reply:
alright, weird ish

Reply:
yes unfortunately postgres only supports timestamps down to microseconds. there is an extension that gives you nanosecond resolution but it seemed more faff to make sure it gets properly installed everywhere. we could have stored them as integers instead of natively as timestamps but it makes querying a bit awkward